spec/views/posts/edit.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
Yeah, these were generated. I'm trying to figure out what to keep and what is junk. I will remove these.